README.md

```diff
@@ -101,12 +101,15 @@ site=https://realm.zulipchat.com
 theme=default
 # Autohide defaults to 'no_autohide', but can be set to 'autohide' to hide the left & right panels except when focused.
 autohide=autohide
-# Footlinks default to 'enabled', but can be set to 'disabled' to hide footlinks.
-footlinks=disabled
+# Maximum footlinks to be shown, defaults to 3, but can be set to any value>=0
+maximum-footlinks=3
```

Comment:
Worth noting equivalent values to 'footlinks', ie include that zero is disabled?